content/java/concepts/math-methods/terms/floor/floor.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'Floor'
+Description: 'Returns the largest integer value that is less than or equal to the argument. When the provided value is either an integer, zero, not a number, or positive or negative infinity.'
+Subjects: 
+    -'Code Foundations'
+    -'Computer Science'
+Tags: 
+    - 'Documentation'
+CatalogContent: 
+  - 'learn-java/static-methods-of-the-math-class'
+---
+
+The static function **`.floor()`** included in the `Math` class returns the largest integer value that is less than or equal to the argument. When the provided value is either an integer, zero, not a number, or positive or negative infinity.
+
+## Syntax
+The Math package can be imported and utilized as shown below
+```java
```

Comment:
```suggestion
```pseudo
```

Let's use \```pseudo code blocks for the "Syntax" section. 😄

Reply:
Made the change as requested.